RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsAlloc.h

```diff
@@ -6,7 +6,7 @@ template <>
 #ifdef __CUDACC__
 void CAHitNtupletGeneratorKernelsGPU::allocateOnGPU(int32_t nHits, cudaStream_t stream) {
 #else
-void CAHitNtupletGeneratorKernelsCPU::allocateOnGPU(int32_t nHits, cudaStream_t stream) {
+inline void CAHitNtupletGeneratorKernelsCPU::allocateOnGPU(int32_t nHits, cudaStream_t stream) {
 #endif
```

Comment:
also, should we add the following for both of these definitions?
```
// NOLINTNEXTLINE(misc-definitions-in-headers)
```

Reply:
Here we could do the same trick as in https://github.com/cms-sw/cmssw/pull/33366#discussion_r621365640, i.e. rename `CAHitNtupletGeneratorKernelsAlloc.h` to `CAHitNtupletGeneratorKernelsAlloc.cc` and in `CAHitNtupletGeneratorKernelsAlloc.cu` have `#include "CAHitNtupletGeneratorKernelsAlloc.cc"`.

Reply:
ah right, updated as suggested